ntfy/cli.py

```diff
@@ -44,16 +46,58 @@ def run_cmd(args):
         *map(int, divmod(time() - start_time, 60))
     )
 
+emojis_hash = {}
+try:
+    emojis_json_path = join(dirname(getfile(run_cmd)),
```

Comment:
Oh wait, you're switching to https://pypi.python.org/pypi/emoji so it doesn't matter.